### ISSUE TITLE
Is the App a medical device?

### ISSUE THREAD

**Issue by tazeroualti** [ROBERT-proximity-tracing/documents#54]
the first question, this app is designed for french people/government, so Why everybody talks English. Is it not more relevant to talk in our mother tongue: French?
Second. has anyone wondered if this app has a medical device purpose? I ask this question because as a product intended for the European market (France) this app must be marked CE before been deployed. To mark it CE you must decide under wich regulation.
In my thought this app is a medical device, more precisely is Software As Medical Device, so it must comply with the EU2017/745 regulation.
Do you have an idea of which EU regulation (except the RGPD) you have to be compliant with?

**Comment by akuckartz** [comment]
A partial reply to the first question from Germany: English is the Lingua Franca in computer science.

**Comment by tazeroualti** [comment]
Sorry I didn't know that German people are involved in this project. I say hello friend and be forgiving about my English.

**Comment by akuckartz** [comment]
> German people are involved in this project

I am not really "involved" in this project (I do not agree with some architectural aspects) but people from several Open Source tracking projects arround the world to some extent are collaborating or at least should do so. The virus is a global matter.

**Comment by vincent-grenoble** [comment]
Hello. Nothing to add for the first question. 

Regarding the second question, I've never heard about it. I gave a look at: [http://eumdr.com/medical-device-rationale/](http://eumdr.com/medical-device-rationale/), as well as Annex XVI, I don't think this is officially speaking a "medical device". If anybody else disagrees, then tell me.

I propose to close  this "Issue" tomorrow unless somebody has some more information (it's a bit marginal to our main topic anyway).

**Comment by beng-git** [comment]
Interesting question. It is an exposure notification / contact tracing application, the medical decision is taken by medical doctors afterwards, and there are no medical decisions taken directly by the device, and moreso I believe it will not be interconnected with any hospital or health agency software, so I'd agree with Vincent.

**Comment by tazeroualti** [comment]
I think that this discussion has to be made and cannot be brushed aside without precise rational. We are also talking about people's health and then have to ensure the safety and performances of the app.
The first step is to accurately describe the intended purpose of the app.
In my understanding, its intended purpose can be summarized as "Warning the user about his potential exposure to the coronavirus". In this case, I think that the App provides some kind of diagnostic that could decide the user take measures as containment, consultation, or what is recommended by the authorities. In other words, the output of the App results in a user special behavior.

The medical device definition is given in the EU regulation 2017/47 in chapter I, Article 2:
‘Medical device’ means any instrument, apparatus, implement, machine, appliance, implant, reagent for in vitro use, software, material or other similar or related article, intended by the manufacturer to be used, alone or in combination, for human beings, for one or more of the specific medical purpose(s) of: 
1.	diagnosis, prevention, monitoring, treatment or alleviation of disease, 
2.	diagnosis, monitoring, treatment, alleviation of or compensation for an injury, 
3.	investigation, replacement, modification, or support of the anatomy or of a physiological process, 
4.	supporting or sustaining life, 
5.	control of conception, 
6.	disinfection of medical devices, 
7.	providing information by means of in vitro examination of specimens derived from the human body; 
and does not achieve its primary intended action by pharmacological, immunological or metabolic means, in or on the human body, but which may be assisted in its intended function by such means.
In this definition, I replaced item dashes by numbers.
I will focus on item 1 and expose my thinking.
- The App does not monitor, treat, or alleviate disease.
- Is the App used for diagnosis? Yes, even if this diagnosis is not based on clinical examination of physiological parameters measurement, the App gives an infection probability, with some interval of confidence.
I feel that the impact of that quantification as a Medical Device would impact a huge of processes and should be anticipated as early as possible.
I don't want to be longer and let you muse over this point.
Greeting

**Comment by vincent-grenoble** [comment]
Let me give some more details regarding my answer above. [http://eumdr.com/medical-device-rationale/](http://eumdr.com/medical-device-rationale/) gives a methodology that I've followed. Do we agree on the following answers?

- Q1: yes
- Q2: yes
- Q3: yes
- Q4: no
- Q5: no
- Q6, looking at [Annex XV1](https://eur-lex.europa.eu/eli/reg/2017/745/2017-05-05#tocId581): not in the list, no

So from my point of view, it's not a medical device under REGULATION (EU) 2017/745. Do I make a mistake somewhere above? Cheers

**Comment by tazeroualti** [comment]
As I saw in this questionnaire: If the answer to questions 1, 2, and 3 is Yes then **the device fulfills the definition of MD** and the regulation likely applies.
For me, this is enough to say that this App is an MD. 
Regards.

**Comment by vincent-grenoble** [comment]
Oups, I've made a mistake. The correct answer is:

- **Q3: no**. This is because "but which may be assisted in its function by such means" does not apply. I've missed this second part of the sentence, sorry.

So the answer to the question if this is a medical device is no and we do not have to care about it.
Thanks for raising the question.

**Comment by tazeroualti** [comment]
I think you made the right answer to question 3. That is a negative question. The answer is YES for the first assertion.
The second assertion "but which may be ..." The word "May" means that this is not mandatory.
At the final, the answer is YES.

**Comment by vincent-grenoble** [comment]
You're right in the sense that "may" does not mean it's mandatory. But if we read the full sentence: 

> Does the product under consideration NOT achieve its principal intended action by pharmacological, immunological or metabolic means, in or on the human body, **but which may be assisted in its function by such means**?

then we can also say that in no case the app "may be assisted by pharmacological, immunological or metabolic means, in or on the human body". It's totally out of scope. So my understanding is that the answer is NO. Do you agree?

**Comment by tazeroualti** [comment]
If you permit me I will remind the text of the regulation in French (I wrote it above in English)

Article 2 Définitions
Aux fins du présent règlement, on entend par:
1) «dispositif médical», tout instrument, appareil, équipement, logiciel, implant, réactif, matière ou autre article, destiné par le fabricant à être utilisé, seul ou en association, chez l'homme pour l'une ou plusieurs des fins médicales précises suivantes:
— diagnostic, prévention, contrôle, prédiction, pronostic, traitement ou atténuation d'une maladie,
— diagnostic, contrôle, traitement, atténuation d'une blessure ou d'un handicap ou compensation de ceux-ci,
— investigation, remplacement ou modification d'une structure ou fonction anatomique ou d'un processus ou état physiologique ou pathologique,
— communication d'informations au moyen d'un examen in vitro d'échantillons provenant du corps humain, y compris les dons d'organes, de sang et de tissus,
**et dont l'action principale voulue dans ou sur le corps humain n'est pas obtenue par des moyens pharmacologiques ou immunologiques ni par métabolisme, mais dont la fonction peut être assistée par de tels moyens.**

Does this persuade you?

**Comment by vincent-grenoble** [comment]
It's surprising to see such an ambiguity in the definition. WRT above text, in our case, "la fonction **ne peut pas** être assistée par de tels moyens". I'm not convinced but we probably need the opinion of an expert in the field.    
Cheers.

**Comment by tazeroualti** [comment]
Yes, I think you should ask an of your expert in this field because the impact on the development life cycle could be very burdensome if you don't take the appropriate measures as early as possible.

**Comment by vincent-grenoble** [comment]
Thank you for raising this point and insisting ;-)

**Comment by tazeroualti** [comment]
All the pleasure is mine, good luck. 
If you have any news, don't hesitate to let me in touch.

**Comment by akuckartz** [comment]
> Following the international health emergency COVID-19, the European Commission and the European Parliament in April 2020 decided to postpone the deadline by one year by postponing the date to 26 May 2021.
https://en.m.wikipedia.org/wiki/Medical_Device_Regulation

Wikipedia provides a reference for that statement, but I do not understand it ;-)

**Comment by tazeroualti** [comment]
This postponement does not affect the status of the App. If the MDR (2017/745) doesn't apply, it will be the MDD (93/42).
For all new project, it is relevant to follow the MDR even if it is postponed.
